relengapi/blueprints/clobberer/__init__.py

```diff
@@ -136,11 +136,31 @@ def branches():
     return [branch[0] for branch in branches]
 
 
+@bp.route('/lastclobber/all', methods=['GET'])
+@apimethod([rest.ClobberTime])
+def lastclobber_all():
+    "Return a sorted list of all clobbers"
+    session = g.db.session(DB_DECLARATIVE_BASE)
+    return session.query(ClobberTime).order_by(ClobberTime.lastclobber)
+
+
+@bp.route('/lastclobber/greater-than/<int:mintime>', methods=['GET'])
+@apimethod([rest.ClobberTime], int)
+def lastclobber_greater_than(mintime):
+    "Return a sorted list of all clobbers greater than the time passed in"
+    session = g.db.session(DB_DECLARATIVE_BASE)
+    return session.query(ClobberTime).filter(
+        ClobberTime.lastclobber > mintime
+    ).order_by(ClobberTime.lastclobber)
+
+
 @bp.route('/lastclobber/branch/by-builder/<string:branch>', methods=['GET'])
 @apimethod({unicode: [rest.ClobberTime]}, unicode)
 def lastclobber_by_builder(branch):
     "Return a dictionary of most recent ClobberTimes grouped by buildername."
-
+    # TODO: simplify this query, new clobberer _only_ tracks max clobber times
+    #  - old clobberer did not - so this query may be simplified under that
+    # assuption
```

Comment:
Let's get a bug or issue on file and point to it in this comment -- sounds like a good fix for a first-time contributor or a sprint.